.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: ruby
+cache: bundler
+sudo: true
+rvm:
+  - 2.2
+env:
+  global:
+    - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
+  matrix:
+    - "RAILS_VERSION=4.2.3"
+services:
+  - redis-server
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install -y clamav libclamav-dev
```

Comment:
may need to add "sudo freshclam"
ref: http://ubuntuforums.org/showthread.php?t=1668311